controllers/dockermachine_controller.go

```diff
@@ -49,20 +52,20 @@ type DockerMachineReconciler struct {
 // +kubebuilder:rbac:groups=cluster.x-k8s.io,resources=clusters;machines,verbs=get;list;watch
 
 // Reconcile handles DockerMachine events
-func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, reterr error) {
-	ctx := context.Background()
-	log := r.Log.WithValues("dockermachine", req.NamespacedName)
+func (r *DockerMachineReconciler) Reconcile(req ctrl.Request) (_ ctrl.Result, rerr error) {
+	ctx := context.TODO()
+	log := log.Log.WithName(machineControllerName).WithValues("dockerMachine", req.NamespacedName)
```

Comment:
for values i've been using skewer-style (css-style) variables, so this should bar `docker-machine` unless you think we should standardize on camel case?

Reply:
I think ideally we should stick with the proper type naming to avoid confusion, so I would suggest `DockerMachine`. That way it is clear that we are referencing a resource of type DockerMachine rather than something else.